schemas/external/repo/directory-page.schema.json

```diff
@@ -0,0 +1,67 @@
+{
```

Comment:
page-object as defined in ACPAPI for directories. This is now separate schema file located in external/repo folder, could also be defined in separate folder like 'hal' links are or moved to some other 'common' location as well ... feedback and proposals welcome.